app/src/main/assets/OpenAPSAMA/determine-basal.js

```diff
@@ -182,9 +182,21 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         return rT;
     }
 
-    // min_bg of 90 -> threshold of 70, 110 -> 80, and 130 -> 90
-    var threshold = min_bg - 0.5*(min_bg-50);
+    // min_bg of 90 -> threshold of 65, 100 -> 70 110 -> 75, and 130 -> 85, or if specified by user, take that value
+ var lgsThreshold = profile.lgsThreshold;
+        var threshold = min_bg - 0.5*(min_bg-40);
```

Comment:
why is default formula changed?

Reply:
Due to a typo. It shouldn't be.

Reply:
I've checked back against openaps/oref0 master, and the min_bg-50 value is incorrect. In openAPS it's min_bg-40. sp the deletion is where the typo is.